src/data/dropdown-data.js

```diff
@@ -0,0 +1,350 @@
+import React from 'react'
```

Comment:
for sample data, better fit them outside of `src/` to not bloat the package size (usually they should reside under `stories/`